enricher/generic/src/test/java/io/jshift/enricher/generic/MavenScmEnricherTest.java

```diff
@@ -45,10 +45,10 @@ public void testMavenScmAll() {
 
         final MavenProject project = new MavenProject();
         final Scm scm = new Scm();
-        scm.setConnection("scm:git:git://github.com/fabric8io/fabric8-maven-plugin.git");
-        scm.setDeveloperConnection("scm:git:git://github.com/fabric8io/fabric8-maven-plugin.git");
+        scm.setConnection("scm:git:git://github.com/fabric8io/jshift-maven-plugin.git");
```

Comment:
I think its better to parameterize using a getName( ) method here. As it will be wrapped up as kubernetes-maven-plugin or okd-maven-plugin or gradle-plugin... @lordofthejars WDYT ?

Reply:
Ignore this.